modules/openapi-generator/src/main/resources/Java/libraries/jersey2/ApiClient.mustache

```diff
@@ -181,6 +186,28 @@ public class ApiClient {
     throw new RuntimeException("No API key authentication configured!");
   }
 
+  /**
+   * Helper method to configure authentications.
+   *
+   * NOTE: This method respects API key aliases using "x-lookup" property
+   *       from OpenAPI specification.
```

Comment:
I don't think this comment will be useful in code. What if you were to change `x-lookup` in the generated templates? Users would be confused because the support is documented in a stale comment.

Reply:
I have removed it.